style.css

```diff
@@ -827,6 +835,7 @@ body {
     height: auto;
     align-items: flex-start;
     justify-content: flex-start;
+    gap: 0.75rem;
   }
 
   .popup-content-container {

```

Comment:
- [ ] Great improvement so far 👏.  As the previous reviewer mentioned, you have put the icons in the buttons (i.e see live and view source buttons) where they are not supposed to be referring to the Figma design. In fact, you have put them in the reverse place. Take a look at the comparison below 👇 
**Your design**
![image](https://user-images.githubusercontent.com/90222110/180574154-bb145775-e5c9-45ce-8c55-6042d7102d75.png)
**Figma**
 ![image](https://user-images.githubusercontent.com/90222110/180574289-f077e5bc-6edf-46a9-bca3-3ff7c22567a5.png)
 Please correct on that 😃 🏃

Reply:
![image](https://user-images.githubusercontent.com/106117541/180584875-9f15e0f6-831a-408b-b122-6742172ce3f2.png)